presto-hive/src/main/java/com/facebook/presto/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -173,6 +164,20 @@ public synchronized void renameDatabase(String databaseName, String newDatabaseN
                 .map(databaseMetadata -> databaseMetadata.toDatabase(databaseName, databaseMetadataDirectory.toString()));
     }
 
+    protected void createDatabase(Database database, boolean checkLocation)
```

Comment:
Why is this needed? Nothing overrides this class (and it's not really designed to be subclassed).